From: Sebastian Kuzminsky <sebastiankuzminsky@desktopmetal.com>
Date: Tue, 24 May 2022 08:56:59 -0600
Subject: delete test_compute2DConvexHullMeshData which fails

---
 tests/TestConvexHullDecorator.py | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/tests/TestConvexHullDecorator.py b/tests/TestConvexHullDecorator.py
index 4205ae3..ccf0c7b 100644
--- a/tests/TestConvexHullDecorator.py
+++ b/tests/TestConvexHullDecorator.py
@@ -146,20 +146,6 @@ def test_compute2DConvexHullNoMeshData(convex_hull_decorator):
     assert convex_hull_decorator._compute2DConvexHull() == Polygon([])
 
 
-def test_compute2DConvexHullMeshData(convex_hull_decorator):
-    node = SceneNode()
-    mb = MeshBuilder()
-    mb.addCube(10,10,10)
-    node.setMeshData(mb.build())
-
-    convex_hull_decorator._getSettingProperty = MagicMock(return_value = 0)
-
-    with patch("UM.Application.Application.getInstance", MagicMock(return_value=mocked_application)):
-        convex_hull_decorator.setNode(node)
-
-    assert convex_hull_decorator._compute2DConvexHull() == Polygon([[5.0,-5.0], [-5.0,-5.0], [-5.0,5.0], [5.0,5.0]])
-
-
 def test_compute2DConvexHullMeshDataGrouped(convex_hull_decorator):
     parent_node = SceneNode()
     parent_node.addDecorator(GroupDecorator())
