Index: Cura/plugins/FirmwareUpdateChecker/FirmwareUpdateChecker.py
===================================================================
--- Cura.orig/plugins/FirmwareUpdateChecker/FirmwareUpdateChecker.py	2020-12-05 19:21:55.076201826 +0100
+++ Cura/plugins/FirmwareUpdateChecker/FirmwareUpdateChecker.py	2020-12-05 19:21:55.072201725 +0100
@@ -30,7 +30,7 @@
 
         # Listen to a Signal that indicates a change in the list of printers, just if the user has enabled the
         # "check for updates" option
-        Application.getInstance().getPreferences().addPreference("info/automatic_update_check", True)
+        Application.getInstance().getPreferences().addPreference("info/automatic_update_check", False)
         if Application.getInstance().getPreferences().getValue("info/automatic_update_check"):
             ContainerRegistry.getInstance().containerAdded.connect(self._onContainerAdded)
 
Index: Cura/plugins/SliceInfoPlugin/SliceInfo.py
===================================================================
--- Cura.orig/plugins/SliceInfoPlugin/SliceInfo.py	2020-12-05 19:21:55.076201826 +0100
+++ Cura/plugins/SliceInfoPlugin/SliceInfo.py	2020-12-05 19:21:55.072201725 +0100
@@ -43,7 +43,7 @@
         self._application = CuraApplication.getInstance()
 
         self._application.getOutputDeviceManager().writeStarted.connect(self._onWriteStarted)
-        self._application.getPreferences().addPreference("info/send_slice_info", True)
+        self._application.getPreferences().addPreference("info/send_slice_info", False)
         self._application.getPreferences().addPreference("info/asked_send_slice_info", False)
 
         self._more_info_dialog = None
